src/main/java/org/jabref/gui/preferences/general/GeneralTab.fxml

```diff
@@ -52,7 +52,7 @@
 
         <CheckBox fx:id="fontOverride" text="%Override default font settings"
                   GridPane.rowIndex="4" GridPane.columnIndex="0"/>
-        <HBox alignment="CENTER_LEFT" spacing="4.0" disable="${!fontOverride.selected}"
```

Comment:
Is there a reason why this was deleted? If not then revert

Reply:
It's my negligence. I'm sorry to cause you trouble!

Reply:
No worries, you are here to learn something and it's part of our mission statement to guide students

Reply:
> No worries, you are here to learn something by contributing to a real open source project and it's part of our mission statement to guide students
> 
> PS: May I ask from which class/university etc you are?

Thank you. I am a student from NEU of China. I found the test failed and I'm fixing it now.